charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -0,0 +1,108 @@
+{{- if .Values.controller.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "argo-cd.controller.fullname" . }}
+  labels:
+    release: {{ .Release.Name }}
```

Comment:
I don't see why we need a new label, especially that it is not in the recommended list: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
We could use `app.kubernetes.io/instance` for selectors

Reply:
Fixed in https://github.com/argoproj/argo-helm/pull/129/commits/89716e2ebc369b56fbf8749df0c304e647156529.  removed `release` label and replaced selectors with `app.kubernetes.io/instance` label.